sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/OneCollectorIngestion.java

```diff
@@ -246,10 +246,12 @@ public void onBeforeCalling(URL url, Map<String, String> headers) {
                 Map<String, String> logHeaders = new HashMap<>(headers);
                 String apiKeys = logHeaders.get(API_KEY);
                 if (apiKeys != null) {
-
-                    /* TODO possibly censor multiple API keys individually instead of whole set */
                     logHeaders.put(API_KEY, HttpUtils.hideSecret(apiKeys));
                 }
+                String tickets = logHeaders.get(TICKETS);
+                if (tickets != null) {
+                    logHeaders.put(TICKETS, HttpUtils.hideSecret(tickets));
```

Comment:
The keys can be useful for debugging. I guess it's better to obfuscate tokens individually.

Reply:
Will try something.